README.md

```diff
@@ -5,19 +5,49 @@
 The CMCD-Validator is a JavaScript library that helps you debug and test HTTP requests with CMCD information as defined in
 [CTA\-5004](https://cdn.cta.tech/cta/media/media/resources/standards/pdfs/cta-5004-final.pdf) in any type of video player.
 
-How to use it:
-- Try it the [demo app for Shaka Player, hls.js, dash.js or Video.js](https://montevideo-tech.github.io/cmcd-validator/)
-- Or install it in your own project `npm install @montevideo-tech/cmcd-validator` 
-
-##  Table  of  contents
-<!-- * [CMCD  Validation  Library  usage](#cmcd-validation-library-usage) -->
-- [CMCD-Validator](#cmcd-validator)
-  - [Table  of  contents](#table--of--contents)
-  - [Repository content](#repository-content)
-  - [Setup](#setup)
-    - [cmcd-validator-library](#cmcd-validator-library)
-    - [cmcd-validator-demo-app](#cmcd-validator-demo-app)
-  - [How to report a bug](#how-to-report-a-bug)
+If you just want to paly with with this library, you can try the [demo app](https://montevideo-tech.github.io/cmcd-validator/) implemented with Video.js, Dash.js, hls.js and Shaka Player.
+
+## Installation
+
+You can use the CDN verison of the validator adding this tag to your document's `<head>`. Then the `window['cmcd-validator-debug']` object will be avaialbe to use it
+```html
```

Comment:
version and available are misspelled. I would remove the it at then end of the sentence and add a final dot.